core/src/test/scala/unit/kafka/server/KafkaConfigTest.scala

```diff
@@ -486,6 +486,8 @@ class KafkaConfigTest {
         case KafkaConfig.MetricNumSamplesProp => assertPropertyInvalid(getBaseProperties, name, "not_a_number", "-1", "0")
         case KafkaConfig.MetricSampleWindowMsProp => assertPropertyInvalid(getBaseProperties, name, "not_a_number", "-1", "0")
         case KafkaConfig.MetricReporterClassesProp => // ignore string
+        case KafkaConfig.RackLocatorClassProp => // ignore
```

Comment:
what/why are we ignoring here? not looking at entire class just seeing the diff hard to say if this makes sense or not to ignore